apps/server/lib/one-drive/src/internal/detect-platform.ts

```
@@ -155,40 +143,24 @@ function getBrowserInfo(): BrowserInfo | null {
-	return "unknown";
+	return arch ? `other:${arch}` : "unknown";
 };
 
 const normalizePlatform = (platform: string): PlatformName => {
-	// Node platforms:
-	// - https://nodejs.org/api/process.html#processplatform
-	// Deno platforms:
-	// - https://doc.deno.land/deno/stable/~/Deno.build
-	// - https://github.com/denoland/deno/issues/14799
-
 	platform = platform.toLowerCase();
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)